src/Assignments/SimpleProblemPage.tsx

```diff
@@ -114,25 +321,76 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
         return <div>{error}</div>;
     }
 
-    if (_.isEmpty(problems)) return <div>There was an error loading this assignment.</div>;
+    // if (_.isEmpty(problems)) return <div>There was an error loading this assignment.</div>;
 
-    if (problems === null || selectedProblemId === null) return <div>Loading...</div>;
+    if (problems === null || selectedProblemId === null) return (
```

Comment:
Our code styling has become inconsistent between the three of us, we should prioritize prettier sooner or later to make sure our code looks the same everywhere